src/Sarif/Baseline/SarifLogBaseliner.cs

```diff
@@ -15,22 +16,27 @@ public SarifLogBaseliner(IEqualityComparer<Result> comparator)
             ResultComparator = comparator;
         }
 
-        public Run CreateBaselinedRun(Run baseLine, Run nextLog)
+        public Run CreateBaselinedRun(Run baseline, Run nextLog)
```

Comment:
Hmm - a bigger problem. I think this class is no longer in use. I think you want to modify SarifLogResultMatcher, and specifically look at SarifLogResultMatcher.BaselineSarifLogs and MatchedResult.CalculateBaselinedResult.